runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -4518,20 +4540,22 @@ J9::X86::TreeEvaluator::VMmonentEvaluator(
    // appropriate excepting instruction we must make sure to reset the
    // excepting instruction since our children may have set it.
    //
+   TR::Compilation *comp = cg->comp();
+   TR_J9VMBase *fej9 = (TR_J9VMBase *)(cg->fe());
    static const char *noInline = feGetEnv("TR_NoInlineMonitor");
-   static int32_t monEntCount = 0;
    static const char *firstMonEnt = feGetEnv("TR_FirstMonEnt");
-   static const char *doCmpFirst = feGetEnv("TR_AddCMPBeforeCMPXCHG");
+   static int32_t monEntCount = 0;
    bool reservingLock = false;
    bool normalLockPreservingReservation = false;
    bool dummyMethodMonitor = false;
-   TR_J9VMBase *fej9 = (TR_J9VMBase *)(cg->fe());
-   TR::Compilation *comp = cg->comp();
+
+   static const char *doCmpFirst = feGetEnv("TR_AddCMPBeforeCMPXCHG");
```

Comment:
Why did this move?

Reply:
I rearranged so that the items that are common for monenter and monexit are in the same order.